src/main/java/sigma/internship/petProject/entity/Game.java

```diff
@@ -0,0 +1,46 @@
+package sigma.internship.petProject.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity(name = "game")
+@Table(name = "game")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+public class Game {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "title", nullable = false, unique = true, length = 30)
+    private String title;
+
+    @Column(name = "description", nullable = false)
+    private String description;
+
+    @Column(name = "is_one_user_action", nullable = false)
+    private boolean isOneUserAction;
+
+    @Column(name = "winning", nullable = false)
+    private double winning;
```

Comment:
It's better use `BigDecimal` type for all values, connected to 'money', instead of `float` and `double`. It's a good practice to use `BigDecimal` to control roundings with a good precision. 
But more information you can read, for example, here https://stackoverflow.com/questions/6320209/javawhy-should-we-use-bigdecimal-instead-of-double-in-the-real-world

You can also easily set up precision and scale for you 'money' values, for example:
```
@Column(name = "winning", precision = 19, scale = 2, nullable = false)
    private BigDecimal winning;
```